network-api/networkapi/buyersguide/models.py

```diff
@@ -93,6 +98,10 @@ def published_product_count(self):
     def __str__(self):
         return self.name
 
+    def save(self, *args, **kwargs):
+        self.slug = slugify(self.name)
```

Comment:
one risk of doing it this way is that the slug can change if we rename a category, so the URL for that category will change. Alternatively, we set the slug once, at object creation (e.g. `if not self.id`) but then of course the slug will no longer reflect the category itself if we rename it.

I suspect we won't be renaming categories all too often though, so this feels... fine? @cadecairos @alanmoo

Reply:
that was sort of the thing we agreed upon when this was added to product names, that they wouldn't need to change. We could try to make it only on create, but if we're changing category names infrequently, I think our redirect middleware should be "good enough" to put in a redirect for a category change.